src/jsx/intrinsic-elements.ts

```diff
@@ -0,0 +1,605 @@
+  dir?: string | undefined
+  draggable?: boolean | undefined
+  hidden?: boolean | undefined
+  id?: string | undefined
+  lang?: string | undefined
+  nonce?: string | undefined
+  placeholder?: string | undefined
+  slot?: string | undefined
+  spellcheck?: boolean | undefined
+  style?: CSSProperties | undefined
+  tabindex?: number | undefined
+  title?: string | undefined
+  translate?: 'yes' | 'no' | undefined
+}
+
+type HTMLAttributeReferrerPolicy =
+  | ''
+  | 'no-referrer'
+  | 'no-referrer-when-downgrade'
+  | 'origin'
+  | 'origin-when-cross-origin'
+  | 'same-origin'
+  | 'strict-origin'
+  | 'strict-origin-when-cross-origin'
+  | 'unsafe-url'
+
+type HTMLAttributeAnchorTarget = '_self' | '_blank' | '_parent' | '_top' | string
+
+interface AnchorHTMLAttributes extends HTMLAttributes {
+  download?: any
+  href?: string | undefined
+  hreflang?: string | undefined
+  media?: string | undefined
+  ping?: string | undefined
+  target?: HTMLAttributeAnchorTarget | undefined
+  type?: string | undefined
+  referrerpolicy?: HTMLAttributeReferrerPolicy | undefined
+}
+
+interface AudioHTMLAttributes extends MediaHTMLAttributes {}
+
+interface AreaHTMLAttributes extends HTMLAttributes {
+  alt?: string | undefined
+  coords?: string | undefined
+  download?: any
+  href?: string | undefined
+  hreflang?: string | undefined
+  media?: string | undefined
+  referrerpolicy?: HTMLAttributeReferrerPolicy | undefined
+  shape?: string | undefined
+  target?: string | undefined
+}
+
+interface BaseHTMLAttributes extends HTMLAttributes {
+  href?: string | undefined
+  target?: string | undefined
+}
+
+interface BlockquoteHTMLAttributes extends HTMLAttributes {
+  cite?: string | undefined
+}
+
+interface ButtonHTMLAttributes extends HTMLAttributes {
+  disabled?: boolean | undefined
+  form?: string | undefined
+  formenctype?: string | undefined
+  formmethod?: string | undefined
+  formnovalidate?: boolean | undefined
+  formtarget?: string | undefined
+  name?: string | undefined
+  type?: 'submit' | 'reset' | 'button' | undefined
+  value?: string | ReadonlyArray<string> | number | undefined
+}
+
+interface CanvasHTMLAttributes extends HTMLAttributes {
+  height?: number | string | undefined
+  width?: number | string | undefined
+}
+
+interface ColHTMLAttributes extends HTMLAttributes {
+  span?: number | undefined
+  width?: number | string | undefined
+}
+
+interface ColgroupHTMLAttributes extends HTMLAttributes {
+  span?: number | undefined
+}
+
+interface DataHTMLAttributes extends HTMLAttributes {
+  value?: string | ReadonlyArray<string> | number | undefined
+}
+
+interface DetailsHTMLAttributes extends HTMLAttributes {
+  open?: boolean | undefined
+  ontoggle?: string | undefined
+}
+
+interface DelHTMLAttributes extends HTMLAttributes {
+  cite?: string | undefined
+  dateTime?: string | undefined
+}
+
+interface DialogHTMLAttributes extends HTMLAttributes {
+  oncancel?: string | undefined
+  onclose?: string | undefined
```

Comment:
I couldn't find both oncancel and onclose attributes
https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement

Reply:
Thank you! There are other `onxxx` attributes which are not necessary for hono-jsx. I've removed them.